src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -769,6 +780,12 @@ export class AppRoot extends polymerElementWithLocalize {
     (this.$.appDrawer as AppDrawerElement).align = alignDir;
     (this.$.sideBar as AppDrawerElement).align = alignDir;
     this.language = language;
+
+    this.shouldShowDocumentationLink = this.hasTranslation('outline-documentation');
+  }
+
+  hasTranslation(key: string) {
+    return this.localize(key) !== key;
```

Comment:
Let's check for empty message as well, so we don't depend on the current behavior.
```suggestion
    const msg = this.localize(key);
    return msg !== key && msg !== '';
```

Reply:
Will this throw if the message needs parameters?